src/main/kotlin/com/sourcegraph/cody/agent/protocol/ContextFile.kt

```diff
@@ -56,15 +64,11 @@ val uriDeserializer =
     }
 
 val uriSerializer = JsonSerializer { uri: URI, type, context ->
-  if (uri == null) {
```

Comment:
Why that change? You seem like you are removing useful safeguard?

Reply:
whoops... good point. intellij suggestion mislead me 

<img width="578" alt="image" src="https://github.com/sourcegraph/jetbrains/assets/19799111/a239e579-314f-4d5b-823f-8cd95bc7d543">